3rd-party/class.jetpack-amp-support.php

```diff
@@ -415,6 +474,16 @@ public static function amp_disable_sharedaddy_css( $enqueue ) {
 		return $enqueue;
 	}
 
+	/**
+	 * Enqueues the AMP specific sharing styles for the sharing icons.
+	 */
+	public static function amp_enqueue_sharing_css() {
+		if ( self::is_amp_request() ) {
+			wp_enqueue_style( 'sharedaddy-amp', plugin_dir_url( dirname( __FILE__ ) ) . 'modules/sharedaddy/amp-sharing.css', array(), JETPACK__VERSION );
+			wp_enqueue_style( 'social-logos' );
```

Comment:
If the new stylesheet depends on `social-logos`, and since that stylesheet is already registered [here](https://github.com/Automattic/jetpack/blob/b96f66a73398cd79dd460127fa07855ee4b85fcf/_inc/social-logos.php#L9-L12), should we just list it as a dependency of the new stylesheet?

Reply:
Good catch!